lib/Runtime/PlatformAgnostic/Platform/Unix/AssemblyCommon.cpp

```diff
@@ -0,0 +1,32 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Common.h"
+#include "PlatformAgnostic/AssemblyCommon.h"
+
+extern void mac_fde_wrapper(const char *dataStart, mac_fde_reg_op reg_op)
+{
+    for(const char *head = dataStart, *last = dataStart + CC_FDE_SIZE;
+        head != last;
```

Comment:
I prefer self-explanatory code instead of comment. `head != last` is meaningless. It is almost always true.